migrate/live-migration/live-migration-faq.md

```diff
@@ -19,6 +19,64 @@ the migration. The logs of these processes can be helpful for troubleshooting
 unexpected behavior. You can find these logs in the `<volume_mount>/logs` directory.
 
 
+## Source database is not running the latest version of TimescaleDB
```

Comment:
Should we phrase like "TimescaleDB extension on source instance is not matching with target"? 

IMHO, The word "latest" is not resonating well here.